docs/developer-docs/latest/development/plugins/i18n.md

```diff
@@ -0,0 +1,377 @@
+    "published_at": "2021-04-07T13:22:46.589Z",
+    "created_at": "2021-04-07T13:22:46.617Z",
+    "updated_at": "2021-04-07T13:22:46.617Z",
+    "closing_period": null,
+    "services": [],
+    "opening_times": [],
+    "dz": [],
+    "cover": null,
+    "images": [],
+    "categories": [],
+    "localizations": []
+}
+```
+
+:::
+::::
+
+## Creating a localization for an existing entry
+
+To create another localization for an existing localized entry, send a POST request to the appropriate URL depending on the type of content:
+
+| Content-Type    | Request URL format                       |
+|:----------------|:-----------------------------------------|
+| Single Type     | `POST /{content-type}/:id/localizations` |  
+| Collection Type | `POST /{content-type}/localizations`     |
+
+
+::: warning CAUTION
+When creating a localization for existing localized entries, the body of the POST request can only accept localized fields.
+:::
+
+When sending the request to a single type, Strapi will:
+
+1. use the `id` as base entry for the non-localized fields and copy them in the new entry
+2. then create a new entry for the given locale and link it with the base entry.
+:::: tabs
+
+::: tab Request
+
+**Example request**
+
+```js
+POST http://localhost:1337/restaurants/8/localizations
+```
+
+**body**
+<!-- * We never used "body" before, see existing entries in Content API. I think it would be a good idea — so it's even more explicit that this code block is the body of the request. If we agree on this, I should update all existing examples in Content API to follow this format. -->
+
+```json
+{
+  "locale": "en",
+  "name": "Restaurant F",
+  "description": "Restaurant facing the sea"
+}
+```
+
+This query:
+
+1. creates a new entry in `en`
+2. links the created entry with `restaurant:8` (they will share the same `localizations` object)
+3. copies every non-localized fields from `restaurant:8` into the new entry and keeps the localized fields from the request's body
+
+:::
+
+::: tab Response
+
+**Example reponse**
+
+```json
+{
+    "id": 9,
+    "name": "Restaurant F",
+    "slug": null,
+    "price_range": null,
+    "contact_email": null,
+    "stars": 1,
+    "averagePrice": null,
+    "address": null,
+    "timestamp": null,
+    "short_description": null,
+    "since": null,
+    "description": "Restaurant facing the sea",
+    "menu": null,
+    "locale": "en",
+    "published_at": "2021-04-08T13:29:53.295Z",
+    "created_at": "2021-04-08T13:29:53.297Z",
+    "updated_at": "2021-04-08T13:29:53.297Z",
+    "closing_period": null,
+    "services": [],
+    "opening_times": [],
+    "dz": [],
+    "cover": null,
+    "images": [],
+    "categories": [],
+    "localizations": [
+        {
+            "id": 8,
+            "locale": "fr",
+            "published_at": "2021-04-07T13:22:46.589Z"
+        }
+    ]
+}
+```
+
+:::
+
+::::
+
+<!-- ? should we also add an example of creating a localization for a collection type? -->
```

Comment:
Do you mean for a single type? (since you've already put an example for a collection type)

Reply:
But yeah an example for a single type could be a good thing

Reply:
@Convly 🤦 well yes. I meant single type — indeed we already POST to the `restaurants` collection type 😅

Reply:
✅ Added example [with this commit](https://github.com/strapi/documentation/pull/224/commits/380a98b55dedf8ad1df4ea27da94ca524ab5f561)